duckduckgo/macos-browser
# add fire animation place holder

Task/Issue URL: https://app.asana.com/0/392891325557410/1200034806153512
Tech Design URL:
CC:

**Description**:

Adds burn animation place holder.  Will scale to fit view size.

**Steps to test this PR**:
1. Open a tab and hit burn.  Animation should play.
1.  Open a few windows and tabs, then hit burn.  Other windows should shut, then animation should play in current view.

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
